modules/statics/src/coins.ts

```diff
@@ -25,10 +25,16 @@ export const coins = CoinMap.fromCoins([
   utxo('tzec', 'Testnet ZCash', Networks.test.zCash, UnderlyingAsset.ZEC),
   account('algo', 'Algorand', Networks.main.algorand, 6, UnderlyingAsset.ALGO),
   account('talgo', 'Testnet Algorand', Networks.test.algorand, 6, UnderlyingAsset.ALGO),
+  account('cgld', 'Celo Gold', Networks.main.celo, 18, UnderlyingAsset.CGLD, ETH_FEATURES),
```

Comment:
There is already coin with this name - this is causing the test failures.

Reply:
We are deprecating CGLD token to use it for Celo. We have a deprecation plan in place (I can add you to it for review if you like)